website/pages-parts/LandingPage/Testimonials.tsx

```diff
@@ -39,17 +46,33 @@ export function Testimonial({
             : styles['border-disable-top-right'],
         )}
       >
-        <h2>{who}</h2>
+        <img src={avatar} alt={who} className={styles['testimonial-avatar']} />
         <p>
-          <b>{company}</b>
+          <b>{who}</b>
+          <br />
+          {position} at <b>{company}</b>
           <br />
-          {position}
         </p>
+        <div className={styles['testimonial-links-wrapper']}>
+          <OvalLink to={linkGithub}>
+            <img
+              alt="github"
+              src="assets/github.svg"
+              className={styles['small-image-icon']}
+            />
+          </OvalLink>
+          <OvalLink to={linkLinkedin}>
+            <img
+              alt="github"
+              src="assets/linkedin.svg"
+              className={styles['small-image-icon']}
+            />
+          </OvalLink>
```

Comment:
1. It should say "GitHub", not "Github"; same for "LinkedIn", not "Linkedin".
2. Instead of using custom icons here, use the `react-feather` package that we use elsewhere.
3. It actually shows that the `OvalLink` component could be a `IconLink` instead.